frontend/src/components/shared/IndicatorReportField.js

```diff
@@ -70,6 +72,22 @@ export default function IndicatorReportField({defaultValue, required, onChange,
 
   const userContext = useContext(UserContext);
 
+  const attriConfig = fullLevelConfig.indicatorReport;
+
+  const attribute2Compass = {
```

Comment:
Also, please make attribute2Compass a property of the Component IndicatorReportField and import it from addEditIndicatorReport, instead of defining it again.

Reply:
Please take a look of CounterFactualField